src/runner/test-run-controller.ts

```diff
@@ -199,6 +211,12 @@ export default class TestRunController extends AsyncEventEmitter {
 
     private async _emitTestRunStart (): Promise<void> {
         await this.emit('test-run-start');
+        if (this._quarantine) {
+            // const { passCount, retryCount } = this._opts.quarantineMode;
+
+            this._quarantine.setFailedQuarantineThreshold(3);
+            this._quarantine.setPassedQuarantineThreshold(1);
```

Comment:
Temporarily hardcoded, while I figure out why `this._opts.quarantineMode` is still just a boolean (instead of including `passCount` and `retryCount` values)